### PR TITLE
Feature - Host builder

### PR DESCRIPTION
**Pull request by CrosRoad95**

### CODE REVIEW

File: SlipeServer.Server/Services/ChatBox.cs [hunk 1]
```diff
@@ -14,10 +14,10 @@ public class ChatBox
     private readonly MtaServer server;
     private readonly RootElement root;
 
-    public ChatBox(MtaServer server, RootElement root)
+    public ChatBox(MtaServer server)
     {
         this.server = server;
-        this.root = root;
+        this.root = server.RootElement;
```
Comment (NanoBob) [inline SlipeServer.Server/Services/ChatBox.cs:20]: Any particular reason for this change?

File: SlipeServer.Server/Resources/Serving/BasicHttpServer.cs [hunk 2]
```diff
@@ -22,8 +22,9 @@ public class BasicHttpServer : IResourceServer
 
     private bool isRunning;
 
-    public BasicHttpServer(Configuration configuration, ILogger logger)
+    public BasicHttpServer(MtaServer mtaServer, ILogger logger)
     {
+        var configuration = mtaServer.Configuration;
```
Comment (NanoBob) [inline SlipeServer.Server/Resources/Serving/BasicHttpServer.cs:27]: Any particular reason for this change?

File: SlipeServer.Server/Configuration.cs [hunk 3]
```diff
@@ -54,7 +54,7 @@ public class Configuration
     /// <summary>
     /// HTTP port to listen to for built-in HTTP server for serving resource files
     /// </summary>
-    public ushort HttpPort { get; set; } = 40680;
+    public ushort HttpPort { get; set; } = 22005;
```
Comment (NanoBob) [inline SlipeServer.Server/Configuration.cs:57]: Please exclude this change from the PR

**Review by NanoBob — CHANGES_REQUESTED** [review]

File: SlipeServer.Server/Services/ChatBox.cs (hunk 1 above)
Comment (CrosRoad95) [reply SlipeServer.Server/Services/ChatBox.cs:20]: Yes, for future multi-mta servers models so it take root element of particular server, not some global one ( it should be done everywhere )

File: SlipeServer.Server/Resources/Serving/BasicHttpServer.cs (hunk 2 above)
Comment (CrosRoad95) [reply SlipeServer.Server/Resources/Serving/BasicHttpServer.cs:27]: The same reason as above: grab configuration from particular mta server, but probably in this case wise would be to allow one http server serve files from multiple mta servers

**Review by NanoBob — APPROVED** [review]

**Pull request merged**